scripts/test-ssl-death-alert.py

```diff
@@ -0,0 +1,73 @@
+# CVE-2016-8610
+# SSL Death Alert
+# OpenSSL SSL/TLS SSL3_AL_WARNING undefined alert flood remote DoS
+
+from __future__ import print_function
+import traceback
+import sys
+
+from tlsfuzzer.runner import Runner
+from tlsfuzzer.messages import Connect, ClientHelloGenerator, \
+        ClientKeyExchangeGenerator, ChangeCipherSpecGenerator, \
+        FinishedGenerator, AlertGenerator
+from tlsfuzzer.expect import ExpectServerHello, ExpectCertificate, \
+        ExpectServerHelloDone, ExpectChangeCipherSpec, ExpectFinished
+from tlslite.constants import CipherSuite, AlertLevel, AlertDescription
+
+
+NUMBER_OF_ALERTS = 4
```

Comment:
I thought that you wanted 2 tests. First, which is not parametrised on command line with constant number of alerts. Second, which is parametrised on command line.
But no problem, I will change this and add second test, but Im not sure about naming these test. `test-ssl-death-alert.py` for this test is ok? And the second just `test-ssl-death-alert-2.py` or something more specific?